package-lock.json

```diff
@@ -348,9 +348,9 @@
       }
     },
     "node_modules/@mdn/browser-compat-data": {
-      "version": "6.0.4",
-      "resolved": "https://registry.npmjs.org/@mdn/browser-compat-data/-/browser-compat-data-6.0.4.tgz",
-      "integrity": "sha512-d9Em7f44DDP1BkJdqBvbjohC25efLgdGoNJvobxm5iMN0EGK677pC83SBjyPmU7OObc9s8Xu+Sv00dzKS+Xdng==",
+      "version": "6.0.5",
```

Comment:
This is still modified even though nothing changed in the repo that would do this.

Consider doing something like `git checkout main -- package-lock.json` or something.